### ISSUE TITLE
Run travis report not working

### ISSUE THREAD

**Issue by WillWcchan** [travis-ci/travis.rb#764]
```
No such file or directory @ rb_sysopen - /Users/Will/.travis/travis.sh/travis.sh
for a full error report, run travis report
Will@Wills-MacBook-Pro ~ % 
Will@Wills-MacBook-Pro ~ % travis report
No such file or directory @ rb_sysopen - /Users/Will/.travis/travis.sh/travis.sh
for a full error report, run travis report
Will@Wills-MacBook-Pro ~ % travis report
No such file or directory @ rb_sysopen - /Users/Will/.travis/travis.sh/travis.sh
for a full error report, run travis report
Will@Wills-MacBook-Pro ~ % ruby -v
ruby 2.7.1p83 (2020-03-31 revision a0c7c23c9c) [x86_64-darwin19]
Will@Wills-MacBook-Pro ~ % 

```

Kept digging and got this:

```
An error occurred running `travis report`:
    Errno::ENOENT: No such file or directory @ rb_sysopen - /Users/Will/.travis/travis.sh/travis.sh
        from /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/fileutils.rb:1386:in `initialize'
        from /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/fileutils.rb:1386:in `open'
        from /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/fileutils.rb:1386:in `block in copy_file'
        from /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/fileutils.rb:1385:in `open'
        from /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/fileutils.rb:1385:in `copy_file'
        from /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/fileutils.rb:492:in `copy_file'
        from /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/fileutils.rb:419:in `block in cp'
        from /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/fileutils.rb:1557:in `block in fu_each_src_dest'
        from /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/fileutils.rb:1566:in `block in fu_each_src_dest0'
        from /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/fileutils.rb:1564:in `each'
        from /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/fileutils.rb:1564:in `fu_each_src_dest0'
        from /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/fileutils.rb:1555:in `fu_each_src_dest'
        from /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/fileutils.rb:418:in `cp'
        from /Users/Will/.gem/ruby/2.6.0/gems/travis-1.10.0.pre.rc1/lib/travis/tools/completion.rb:34:in `update_completion'
        from /Users/Will/.gem/ruby/2.6.0/gems/travis-1.10.0.pre.rc1/lib/travis/cli/command.rb:177:in `block in check_completion'
        from /Users/Will/.gem/ruby/2.6.0/gems/travis-1.10.0.pre.rc1/lib/travis/cli/command.rb:117:in `write_to'
        from /Users/Will/.gem/ruby/2.6.0/gems/travis-1.10.0.pre.rc1/lib/travis/cli/command.rb:176:in `check_completion'
        from /Users/Will/.gem/ruby/2.6.0/gems/travis-1.10.0.pre.rc1/lib/travis/cli/command.rb:198:in `execute'
        from /Users/Will/.gem/ruby/2.6.0/gems/travis-1.10.0.pre.rc1/lib/travis/cli.rb:64:in `run'
        from /Users/Will/.gem/ruby/2.6.0/gems/travis-1.10.0.pre.rc1/bin/travis:18:in `<top (required)>'
        from /usr/local/bin/travis:23:in `load'
        from /usr/local/bin/travis:23:in `<main>'
```

Don't know what happened.

**Comment by BanzaiMan** [comment]
`git bisect` reveals that https://github.com/travis-ci/travis.rb/commit/36ba42a2dfea307bb123e37d83c8539767470b89 is the first bad commit.

**Comment by BanzaiMan** [comment]
https://github.com/travis-ci/travis.rb/commit/fb95c5a757c6649a8ed4025d6e04680858b426a5 should fix the problem. I plan to make a new pre-release (rc3), and once we have a few tests done, a 1.10.

**Comment by BanzaiMan** [comment]
https://rubygems.org/gems/travis/versions/1.10.0.pre.rc3

**Comment by BanzaiMan** [comment]
```sh-session
travis@bf8e99356098:~$ gem install travis --pre
Fetching i18n-1.8.5.gem
Fetching faraday-1.0.1.gem
Fetching faraday_middleware-1.0.0.gem
Fetching highline-2.0.3.gem
Fetching concurrent-ruby-1.1.7.gem
Fetching tzinfo-1.2.7.gem
Fetching multipart-post-2.1.1.gem
Fetching thread_safe-0.3.6.gem
Fetching activesupport-5.2.4.4.gem
Fetching multi_json-1.15.0.gem
Fetching public_suffix-4.0.6.gem
Fetching addressable-2.7.0.gem
Fetching net-http-persistent-2.9.4.gem
Fetching net-http-pipeline-1.0.1.gem
Fetching gh-0.18.0.gem
Fetching travis-1.10.0.pre.rc3.gem
Fetching launchy-2.4.3.gem
Fetching json_pure-2.3.1.gem
Fetching websocket-1.2.8.gem
Fetching pusher-client-0.6.2.gem
Successfully installed multipart-post-2.1.1
Successfully installed faraday-1.0.1
Successfully installed faraday_middleware-1.0.0
Successfully installed highline-2.0.3
Successfully installed concurrent-ruby-1.1.7

HEADS UP! i18n 1.1 changed fallbacks to exclude default locale.
But that may break your application.

If you are upgrading your Rails application from an older version of Rails:

Please check your Rails app for 'config.i18n.fallbacks = true'.
If you're using I18n (>= 1.1.0) and Rails (< 5.2.2), this should be
'config.i18n.fallbacks = [I18n.default_locale]'.
If not, fallbacks will be broken in your app by I18n 1.1.x.

If you are starting a NEW Rails application, you can ignore this notice.

For more info see:
https://github.com/svenfuchs/i18n/releases/tag/v1.1.0

Successfully installed i18n-1.8.5
Successfully installed thread_safe-0.3.6
Successfully installed tzinfo-1.2.7
Successfully installed activesupport-5.2.4.4
Successfully installed multi_json-1.15.0
Successfully installed public_suffix-4.0.6
Successfully installed addressable-2.7.0
Successfully installed net-http-persistent-2.9.4
Successfully installed net-http-pipeline-1.0.1
Successfully installed gh-0.18.0
Successfully installed launchy-2.4.3
Successfully installed json_pure-2.3.1
Successfully installed websocket-1.2.8
Successfully installed pusher-client-0.6.2
Successfully installed travis-1.10.0.pre.rc3
20 gems installed
travis@bf8e99356098:~$ travis history -r travis-ci/travis.rb
Shell completion not installed. Would you like to install it now? |y|
#1024 passed:    master Merge pull request #678 from travis-ci/sf-fix-file-encryption-var-names
#1023 passed:    master (PR #653) Install Travis CI Client on Mac OS X via Homebrew
#1022 passed:    master Merge pull request #678 from travis-ci/sf-fix-file-encryption-var-names
#1021 passed:    master Merge pull request #678 from travis-ci/sf-fix-file-encryption-var-names
#1020 passed:    master Merge pull request #678 from travis-ci/sf-fix-file-encryption-var-names
#1019 errored:   master Bump version to 1.8.10
#1018 passed:    master (PR #678) Remove Ruby 1.9.2 from the build matrix
#1017 passed:    sf-fix-file-encryption-var-names Remove Ruby 1.9.2 from the build matrix
#1016 errored:   master (PR #678) make sure encrypted file variable names are unique per filename
#1015 errored:   sf-fix-file-encryption-var-names make sure encrypted file variable names are unique per filename
```
